cli/azd/pkg/tools/github/github.go

```diff
@@ -323,3 +377,185 @@ func isEnvVarTokenSource(message string) bool {
 
 	return false
 }
+
+func extractFromZip(src, dst string) (string, error) {
+	zipReader, err := zip.OpenReader(src)
+	if err != nil {
+		return "", err
+	}
+
+	defer zipReader.Close()
+
+	var extractedAt string
+	for _, file := range zipReader.File {
+		if !file.FileInfo().IsDir() && strings.Contains(file.Name, "gh") {
```

Comment:
Do you think we could possibly get into trouble by not extracting other files in the archive? I feel like `gh` expects users to install this by unzipping the entire archive, but I think you know more about this than I do.

Reply:
Again, it's a statically linked binary with no other asset files, but curious if that could change if they decided to create other file resource assets.

Reply:
Dumping the archive into `bin` feels wrong too. I think this is fine for now.